lib/osf-components/addon/components/institutions-widget/template.hbs

```diff
@@ -0,0 +1,44 @@
+<div data-test-institutions-widget>
+    <div local-class="InstitutionWidget__title">
+        {{t "osf-components.institutions-widget.affiliated_institutions"}}
+        {{#if (not this.readOnly)}}
+            <button 
+               aria-label={{t "general.add"}} 
+               class="add" 
+               onclick=
```

Comment:
I think we've decided to use Ember actions everywhere instead of DOM event handlers, so please omit `onclick=` here and for the buttons below.